componentLibraries/defaultLibrary/Hydraulic/Valves/PressureControlValves/HydraulicOverCenterValve.hpp

```diff
@@ -80,8 +80,8 @@ namespace hopsan {
             addOutputVariable("xv", "Equivalent spool position", "", &mpXv);
 
             addConstant("tao", "Time Constant of Spool", "s", 0.01, mTao);
-            addConstant("k_cs", "Steady State Characteristic due to Spring", "(m^3/s)/Pa", 0.00000001, mKcs);
```

Comment:
I am not sure if it makes sense to call it LeakageCoefficient in these cases (valves), but since the unit is the same that is what it will get labeled as anyway. What do you thing @robbr48

Reply:
I am not really sure what quantity this parameter should have. It is technically behaving as a leakage, but the pressure in these parameters might not always be the pressure drop. 

Actually I think unit conversion should only work when you specify a quantity. If a specific unit is given, it should not automatically try to identify the quantity. That makes sense only for trivial units and quantities. I know we discussed this before, with "rad/s" being both a frequency and angular velocity unit. But I guess this would require a significant amount of re-coding.

Reply:
Turning of the automatic lookup of quantity should be fairly easy, but it has to be done in a couple of places.
Maybe later....